scripts/create-release-pr.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+if [ -z "${1:-}" ]
+then
+  echo "Error: Missing required positional argument: patch|minor|major"
+  exit 1
+fi
+
+function push_version_bump() {
+  local newVersion
+  newVersion=$(node -p 'require("./lerna.json").version')
+
+  git checkout -b "${newVersion}"
```

Comment:
I'd suggest avoiding creating branches with the same name as tags. This creates ambiguous git references, and makes it difficult to select a branch or tag specifically with some commands (you have to give the full path, e.g. `refs/tags/vX.Y.Z`).

This is the convention I've been using:
```suggestion
  git checkout -b "release-${newVersion}"
```

Reply:
I see that this will need to be referenced on line 20 as well, so this suggestion won't suffice on its own.

Reply:
Fixed here: 92e290a